src/components/MuiVirtualizedTable/MuiVirtualizedTable.js

```diff
@@ -13,14 +13,23 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import TableCell from '@mui/material/TableCell';
 import { AutoSizer, Column, Table } from 'react-virtualized';
-import TableSortLabel from '@mui/material/TableSortLabel';
 import memoize from 'memoize-one';
 import GetAppIcon from '@mui/icons-material/GetApp';
 import CsvDownloader from 'react-csv-downloader';
 import IconButton from '@mui/material/IconButton';
 import { FormattedMessage } from 'react-intl';
 import withStyles from '@mui/styles/withStyles';
 import OverflowableText from '../OverflowableText/overflowable-text';
+import {
+    collectibleHelper,
+    getHelper,
+    KeyedColumnsRowIndexer,
+    CHANGE_WAYS,
+} from './KeyedColumnsRowIndexer';
+import Popover from '@mui/material/Popover';
+import Autocomplete from '@mui/material/Autocomplete';
+import { Chip, TextField } from '@mui/material';
+import { ColumnHeader } from './ColumnHeader';
```

Comment:
If you had a default export you could remove those {} when importing ColumnHeader component